exporter/datasetexporter/README.md

```diff
@@ -34,13 +42,25 @@ If you do not want to specify `api_key` in the file, you can use the [builtin fu
   - `retry_max_elapsed_time` (default = 300s): Is the maximum amount of time spent trying to send a buffer.
 - `logs`:
   - `export_resource_info_on_event` (default = false): Include resource info to DataSet Event while exporting Logs. This is especially useful when reducing DataSet billable log volume.
+- `server_host`:
+  - `server_host` (default = ''): Use the specified value as server host for the event.
```

Comment:
May want to clarify it and say for the value for all the events.

Reply:
Fixed